doc/scenegraph.dox

```diff
@@ -65,8 +65,9 @@ usually a transformation matrix. The whole scene is rendered using a camera
 like field-of-view, aspect ratio and near/far clipping planes. The final
 projective object transform @f$ \boldsymbol{M_i} @f$, relative to camera, is
 calculated as a combination of all relative transformations up to the scene
-root (an absolute transformation), multiplied by an inverse of camera absolute
-transformation. For the object @f$ o_3 @f$ it's as follows: @f[
+root (an absolute transformation), multiplied by the inverse of the camera's absolute
+transformation. For the object @f$ o_3 @f$ its final transform @f$ \boldsymbol{M_3} @f$
+is produced as follows: @f[
```

Comment:
Here, I tried to get across the purpose of the equation included below. Found here: https://doc.magnum.graphics/magnum/scenegraph.html#scenegraph-basic-concepts